client/src/views/util.js

```diff
@@ -8,13 +8,17 @@ const DEFAULT_PRECISION = 0
 
 const pad = n => n < 10 ? '0'+n : n
 
-export const formatTime = unix => {
+export const formatTimezone = unix => {
```

Comment:
I would consider removing the `export` now that `formatTimezone` is only used internally by `formatTime` and not by anything external.

Reply:
RE: The default still needs to change here. (Should I change the GMT in the formatTime to UTC)?

Reply:
I meant changing the `with_tz` argument to default to true ([rational explained here](https://github.com/Blockstream/esplora/pull/261#discussion_r504283261)).

But yes, you could also change `formatTimezone` to return `UTC` instead of `GMT` when the timezone offset is 0, which would be more accurate.

Reply:
I added (with_tz = true) to the function's argument

Reply:
I meant to use `UTC` only when the timezone offset is 0, i.e.

```js
return `${tzOffset == 0 ? 'UTC' : 'GMT ' + (tzOffset < 0 ? '' : '+') + (tzOffset/60)}`
```

Reply:
Updated.